### ISSUE TITLE
Unable to download file if it has a suspiciously HTML filename

### ISSUE THREAD

**Issue by pdehaan** [mozilla/send#213]
Found in https://send.dev.mozaws.net/__version__ (8d00372):

```js
{
    "commit": "8d00372",
    "source": "https://github.com/mozilla/send/",
    "version": "0.1.2"
}
```

---

### Steps to reproduce:
1. In Finder, rename a file to something like: `<script>alert(1)</script>.gif`.
1. Upload said file into Firefox Send.
1. Try and download the file.

### Actual results:
Interestingly, when I try and upload the file, it gets "cleaned" to `<script>alert(1)<:script>.gif` in the `<table>`.

#### After uploading:

<img width="1440" alt="firefox_send" src="https://user-images.githubusercontent.com/557895/28304514-d9bde5e6-6b4c-11e7-87c1-49b1f5432375.png">

**Figure 1:** Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src https://send.dev.mozaws.net https://ssl.google-analytics.com”). Source: alert(1)<:script>.gif.

#### On download page:

<img width="1440" alt="download_your_file" src="https://user-images.githubusercontent.com/557895/28304583-1a315bbc-6b4d-11e7-8cbf-d06bada67bc6.png">

**Figure 2:** TypeError: document.getElementById(...) is null

**Comment by abhinadduri** [comment]
I've fixed the download issue, but I'm not sure why the '/' gets cleaned to a ':'.

**Comment by dannycoates** [comment]
fixed in #232
